app/controllers/popup_controller.rb

```diff
@@ -46,9 +46,9 @@ def team_users_popup
       # However, we want their user_id. This is not possible for teams, so we just return the current id
       reviewer_id = ResponseMap.find(params[:id2]).reviewer_id
       if @assignment.reviewer_is_team
-        @reviewer_id = Participant.find(reviewer_id).user_id
-      else
```

Comment:
Was done in response to a suggestion by Sanket to fix the breakage of the popup page.